.travis.yml

```diff
@@ -27,11 +19,33 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
-
+          env: DEPENDENCIES="minimun" COVERAGE=true TEST_COMMAND="composer test-ci"
+
+          # Test the latest stable release
+        - php: 7.1
+
+          # Test LTS versions
+        - php: 7.1
+          env: DEPENDENCIES="symfony/lts:v2"
```

Comment:
clever use of the package, great discovery!

Reply:
What if some bundle authors actively only want to support 2 Symfony versions at one time? What's what we're talking about in FOSUserBundle https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2639#issuecomment-344907215

Reply:
This can still install a mixed set of 2.7 and 2.8 components. What about using `symfony/symfony:v2.8.*` which will force any symfony component to be in it's v2.8 version?

Reply:
Sorry, I realize I was unclear! What I *really* mean was: in FOSUserBundle, they only want to support Symfony 3 and Symfony 4 (and then perhaps just do bug-fixes fro 2.7/2.8 branches).

In that situation, I guess they would just remove this *one* line and change the minimum dependencies matrix to 5.5 (from 5.4)?

Reply:
remove the Symfony 2 LTS job and bump your min requirement

Reply:
re requiring symfony/symfony: that hides problems if we miss depending on some symfony components. if we do that, we should require each component we use in the configured version instead. (which duplicates composer.json, so still not very nice)

Reply:
We still have the "Test the latest stable release" job which does not alter the `composer.json`.